index.js

```diff
@@ -4,7 +4,49 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+
+//const parameters = process.argv;
+let i;
+let result = 0;
+function calculator() {
+   
+if (process.argv.length == 2 ) {
+    return "No operation provided..."
+}
+if (process.argv[2] === "modulo"){
+   return "Invalid operation: modulo"
+}
+
+if (process.argv.length == 3) {
+    if ((process.argv[2] === "plus") || (process.argv[2] === "minus")) {
+        console.log("No numbers provided...") 
+        return "No numbers provided..."}
+    
+}
+if (process.argv[2]  == "plus") {
+    result = 0;
+    for (i=3; i < process.argv.length ; i++ ) {
+    result = result + Number(process.argv[i]);
+    }
+    console.log(result)
+    return result 
+}
+
+if (process.argv[2] == "minus") {
+    result = process.argv[3];
+    for (i=4; i < process.argv.length ; i++ ) {
+    result = result - Number(process.argv[i]); 
+    }
+    console.log(result)
+    return result
+}   
+
+}
+//console.log(result)
+
```

Comment:
Be careful with your formatting. While it doesn't impact the functionality of your code, it helps readability. Code inside code blocks(ifs, for loops) should be indented once